aboutme.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<meta name="viewport" content="width=device-width, initial-scale=1.0">
+		<link rel="stylesheet" href="portfolio.css">
+		<title>Portfolio</title>
+	</head>
+	<body class="background">
+		<nav>
+			 <div class="name">
+		        <ul>
+					<li>Isabelle Cochrane</li>
+				</ul>
+			</div>
+			<div class="navlinks">
+		<ul>
+		    <li><a href=portfolionav.html>Projects</a></li>
+		    <li><a href=aboutme.html>About Me</a>
+		    </li>
+		    <li><a href=contact.html>Contact</a></li>
+		  </ul>
+		 </div>
+		</nav>
+		<div class="aboutcontent">
+			<h1>About Me</h1>
+			<p>blahblahblah lahblahblah lahblahblah lahblahblah lahblahblah lahblahblah lahblahblah lahblahblah lahblahblah lahblahblah lahblahblah lahblahblah lahblahblah lahblahblah lahblahblah lahblahblah lahblahblah lahblahblah lahblahblah lahblahblah lahblahblah lahblahblah lahblahblah lahblahblah lahblahblah
```

Comment:
This P is very narrow for me, I think there's a width set on the P tag (of 200px) that has caused this to break, probably from combining your stylesheets.

Reply:
ah yes I will change the format to specify that the 200px is for the portfolio page p